app/mail/DefaultMails.scala

```diff
@@ -84,7 +84,6 @@ class DefaultMails @Inject()(conf: WkConf) {
                  organizationName: String,
                  senderName: String): Mail = {
     val host = Try { new URL(uri) }.toOption.getOrElse(uri)
-    println(s"Invite mail to $recipient with link $uri/invite/$inviteTokenValue")
```

Comment:
Re-add this line for testing => enables finding the invite link needed for testing in the console

Reply:
by the way, you can also set `mail.logToStdout = true` in `application.conf` to see the mail contents in logging